lib/stat_tracker.rb

```diff
@@ -1,14 +1,9 @@
 require 'CSV'
 
 class StatTracker
-  
-  attr_reader :data
-  def self.from_csv(data)
-    StatTracker.new(data)
+  def self.from_csv(object, path)
+    CSV.foreach(path, headers: true, header_converters: :symbol) do |row|
+      object.add(object.new(row))
+    end
   end
-
-  def initialize(data)
```

Comment:
I see you got rid of initialize. Mind explaining why? Not that I don't approve, just wanna know why you did. I've never had to do that

Reply:
I thought if I had to instantiate  everything within the stat tracker I'd end up with too much so I just created a separate team class and plan to create the additional necessary classes for future methods as well.